components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/fetch/GeckoViewFetchClient.kt

```diff
@@ -107,11 +109,16 @@ private fun WebRequest.Builder.addBodyFrom(request: Request): WebRequest.Builder
     return this
 }
 
-private fun WebResponse.toResponse(): Response {
+@VisibleForTesting
+internal fun WebResponse.toResponse(): Response {
     val headers = translateHeaders(this)
+    val isBlobUrl = uri.toUri().scheme == "blob"
+    // Blobs use the same API as HTTP requests, but they don't send us the same metadata.
+    // If don't get any exceptions we assumed that everything went well.
```

Comment:
nit: If no exception is thrown we assume success"

Reply:
👍